README.md

```diff
@@ -42,6 +44,57 @@ use_frameworks!
 
 Then run `pod install` with CocoaPods 0.36 or newer.
 
+## Adding Runes as a Framework Dependency ##
+
+Runes can also be used as a low-overhead dependency for defining these
+operators for your own types. After linking to `Runes.swift`, you will be able
+to define implementations for your types, and your users will be able to
+import your framework to use these operators without needing to link against
+Runes directly.
+
+In order to use Runes as a framework dependency, you're going to want to use
+Git Submodules. This will allow Carthage and CocoaPods to pull down the Runes
+source code without needing to expose this to your users.
```

Comment:
I found this section confusing ... although I don't know what to change. Once I figured out what you were saying it made total sense.